spec/models/repository_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 RSpec.describe Repository, type: :model do
   describe 'validations' do
-    %i[name visibility repo_id web_url].each do |attribute|
+    %i[name visibility repo_id web_url namespace_path].each do |attribute|
```

Comment:
This is one of the drawbacks of writing DRY tests. _Additions_ to code should result in _additions_ to tests rather than _modifications_